pkg/proxy/userspace/proxysocket.go

```diff
@@ -129,12 +134,14 @@ func (tcp *tcpProxySocket) ProxyLoop(service proxy.ServicePortName, myInfo *serv
 				// Then the service port was just closed so the accept failure is to be expected.
 				return
 			}
+			proxier.recorder.Eventf(proxier.nodeRef, "TooManyFDsError", "TCP: ProxyLoop Accept failed")
```

Comment:
This is not a tooManyFDsError here, also needs more info (the err), also about Service or endpoints